pkg/controller/usersignup/usersignup_controller.go

```diff
@@ -5,7 +5,7 @@ import (
 	"crypto/md5"
 	"encoding/hex"
 	"fmt"
-	"regexp"
+	"github.com/codeready-toolchain/toolchain-common/pkg/usersignup"
```

Comment:
could you move this import declaration to line 11, with all the other `github.com/codeready-toolchain/...` ones?

Reply:
fixed here: 16c3c1b7a8c33fe7504c3f58a757a168b7acef05

Reply:
thanks ;)